docs/tutorials/index.md

```diff
@@ -71,17 +70,19 @@ The distribution directory contains `LICENSE` and `NOTICE` files and subdirector
 ## Start up Druid services
 
 Start up Druid services using the `auto` single-machine configuration.
-This configuration includes default settings that are appropriate for this tutorial, such as loading the `druid-multi-stage-query` extension by default so that you can use the MSQ task engine.
+This configuration includes default settings that are appropriate for this tutorial.
+For example, it loads `druid-multi-stage-query` extension so that you can use the MSQ task engine to ingest data using SQL.
 
-You can view that setting and others in the configuration files in the `conf/druid/auto`. 
+You can view the default settings in the configuration files located in `conf/druid/auto`.
 
 From the apache-druid-{{DRUIDVERSION}} package root, run the following command:
 
 ```bash
 ./bin/start-druid
 ```
 
-This brings up instances of ZooKeeper and the Druid services and may use up to 80% of the total available system memory. To explicitly set the total memory available to Druid, pass a value for the memory parameter, e.g. `./bin/start-druid -m 16g` or `./bin/start-druid --memory 16g`.
+This launches instances of ZooKeeper and the Druid services.
```

Comment:
Suggestion: `start-druid` launches ZooKeeper and the various Druid services.

Does this cluster use the embedded Derby DB? If so, we might want to say that and note that a production system would use MySQL or Postgres instead.